src/application/App.tsx

```diff
@@ -74,22 +74,28 @@ export const App = () => {
     devtoolsMachine.provide({
       actions: {
         resetStore: () => {
-          apolloClient.resetStore().catch(() => {});
+          apolloClient.clearStore().catch(() => {});
```

Comment:
Because this now uses `clearStore` instead of `resetStore`, active queries no longer clear out their data. This introduces a small regression where refreshing the page will show the stale data in the app until a new client is registered.

Unfortunately I've been unable to get the mounted queries to respond to cache updates after the cache has been cleared, so the data remains on screen. The `refetch` after the first client is registered will fix this issue.